package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "google-gax",
-  "version": "0.5.0",
+  "version": "0.5.2",
```

Comment:
I previously pushed 0.5.1 to npmjs.com (see https://www.npmjs.com/package/google-gax/) without committing the change of the version.  I don't remember why exactly (sorry), but probably that was a minor typo fix or something found in 0.5.0 after I pushed.